earn/src/data/Uniswap.ts

```diff
@@ -18,6 +23,16 @@ export type UniswapPosition = {
 
 export type UniswapPositionPrior = Omit<UniswapPosition, 'liquidity'>;
 
+export type UniswapNFTPosition = {
+  operator: string;
+  token0: Token;
+  token1: Token;
+  fee: number;
+  tickLower: number;
+  tickUpper: number;
```

Comment:
My personal convention is to just call these `lower` and `upper`

Reply:
I was following the naming convention from the contract.